test/runtime/Policies.go

```diff
@@ -1393,3 +1394,101 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(res.Output().String()).Should(ContainSubstring(allowedVerdict), "Policy trace did not contain %s", allowedVerdict)
 	})
 })
+
+var _ = Describe("RuntimeValidatedPolicyDropAllTests", func() {
+	var once sync.Once
+	var logger *logrus.Entry
+	var vm *helpers.SSHMeta
+	var dropAll bool
+
+	initialize := func() {
+		logger = log.WithFields(logrus.Fields{"test": "RuntimeValidatedPolicyDropAllTests"})
+		logger.Info("Starting")
+		vm = helpers.CreateNewRuntimeHelper(helpers.Runtime, logger)
+		areEndpointsReady := vm.WaitEndpointsReady()
+		Expect(areEndpointsReady).Should(BeTrue())
+	}
+
+	BeforeEach(func() {
+		once.Do(initialize)
+		vm.PolicyDelAll()
+		res := vm.SetPolicyEnforcement(helpers.PolicyEnforcementDefault)
+		res.ExpectSuccess("Setting policyEnforcement to default")
+	})
+
+	AfterEach(func() {
+		vm.ValidateNoErrorsOnLogs(CurrentGinkgoTestDescription().Duration)
+		if CurrentGinkgoTestDescription().Failed {
+			vm.ReportFailed()
+		}
+	})
+
+	It("DROP_ALL Ingress Policy test", func() {
+		By("Starting cilium monitor in background to filter DROP requests")
+		ctx, cancel := context.WithCancel(context.Background())
+		res := vm.ExecContext(ctx, "cilium monitor --type drop")
+		defer cancel()
+
+		By("Create an endpoint with no labels to test DROP_ALL")
+		cmdStr := "docker run -dt --name ingressContainer --net=cilium-net tgraf/netperf"
+		vm.Exec(cmdStr).ExpectSuccess("Failed to create docker container")
+
+		endpointCmd := "cilium endpoint list | grep \"waiting-for-identity\" | awk '{ print $1}'"
+		endpointID := strings.TrimSpace(vm.Exec(endpointCmd).GetStdOut())
+		endpointCmd = "cilium endpoint list | grep \"waiting-for-identity\" | awk '{ print $7}'"
+		endpointIP := strings.TrimSpace(vm.Exec(endpointCmd).GetStdOut())
+
+		By("Testing endpoint lxc_config.h for DROP_ALL")
+
+		lxcCmd := fmt.Sprintf("sudo cat /var/run/cilium/state/%s/lxc_config.h | grep 'DROP_ALL'", endpointID)
+
+		dropAll = false
+		if strings.TrimSpace(vm.Exec(lxcCmd).GetStdOut()) == "#define DROP_ALL" {
+			dropAll = true
+		}
+
+		Expect(dropAll).Should(BeTrue(), "DROP_ALL failed for ingress. lxc_config.h does not contain the DROP_ALL macro")
+
+		By("Testing ingress with ping from localhost to endpoint with no labels")
+		lxcCmd = fmt.Sprintf("ping %s", endpointIP)
+		vm.Exec(lxcCmd)
+
+		By("Testing cilium monitor drop")
+		err := res.WaitUntilMatch("drop (Policy denied (L3))")
```

Comment:
I wonder if we could reuse the actual string from the monitor package?

https://github.com/cilium/cilium/blob/0d745b8acf29139e88fe1fb7d4f9c0441082d158/pkg/monitor/datapath_drop.go#L47

(I'd like to rename it at some point, this would make it easier)

Reply:
`pkg/monitor` has nothing to do with BPF compilation.

Reply:
@joestringer true. In the past though I have found surprising transitive dependencies ;)